README.md

```diff
@@ -49,6 +49,24 @@ Download the latest release [https://github.com/andredumas/techan.js/releases/la
 npm install --save techan
 ```
 
+#### Webpack (ES6)
+```javascript
+// webpack.config.js
+config = {
+    ...,
+    resolve: {
+        ...,
+        alias: {
+            'techan': path.resolve('node_modules/techan/src/techan.js')
+        }
+    }
+}
+
+// usage
+import * as d3 from 'd3';
```

Comment:
Okay, so this doesn't work as expected with anything requiring mouse events (eg crosshairs). https://github.com/d3/d3/issues/2733

I'm still searching for a solution that lets ``techan.js``'s events work with babel. Do you perhaps have any insights?